torch/onnx/symbolic_opset9.py

```diff
@@ -4683,11 +4683,11 @@ def linalg_norm(
 @symbolic_helper.parse_args("v", "f", "is", "i", "v")
 def linalg_vector_norm(
     g,
-    self: _C.Value,
+    self: torch._C.Value,
     ord: float,
-    dim: typing.List[int],
+    dim: List[int],
     keepdim: int,
-    dtype: _C.Value,
+    dtype: torch._C.Value,
```

Comment:
Just a small reminder about removing torch. in _C types. Other than that lgtm. Thanks for the changes!